pkg/util/util.go

```diff
@@ -202,7 +221,7 @@ func GetBrokerConfig(broker v1beta1.Broker, clusterSpec v1beta1.KafkaClusterSpec
 
 	err := mergo.Merge(bConfig, clusterSpec.BrokerConfigGroups[broker.BrokerConfigGroup], mergo.WithAppendSlice)
 	if err != nil {
-		return nil, errors.WrapIf(err, "could not merge brokerConfig with ConfigGroup")
+		return nil, emperror.WrapIf(err, "could not merge brokerConfig with ConfigGroup")
```

Comment:
is this change required?

Reply:
good catch! I removed the redundant changes.